tests/data_context/test_data_context_in_code_config.py

```diff
@@ -119,234 +119,6 @@ def list_s3_bucket_contents(bucket: str, prefix: str) -> Set[str]:
     }
 
 
-@pytest.mark.aws_deps
-@mock_s3
-def test_DataContext_construct_data_context_id_uses_id_of_currently_configured_expectations_store(
-    aws_credentials,
```

Comment:
We no longer have this conditional logic based on expectations store - we don't need these tests